signhost.php

```diff
@@ -230,73 +222,49 @@ class Signer implements JsonSerializable {
 
 	function __construct(
```

Comment:
Again, I think we accept the compatibility break for now?

Reply:
Yes, since we are not using SemVer breaking changes are okay.